api/src/routers/future-meals.js

```diff
@@ -0,0 +1,17 @@
+import express from "express";
+import knex from "../database_client.js";
+
+const futureMealsRouter = express.Router();
+
+
+futureMealsRouter.get('/', async (req, res) => {
+    try {
+        const now = new Date();
+        const future_meals =  await knex('Meal').select('*').where('when', '>', now);
```

Comment:
I think this will do to:
```
const future_meals =  await knex('Meal').where('when', '>', now);
```
`select` is the default, so we do not necessarily need it